backend/src/test/java/botobo/core/acceptance/user/UserAcceptanceTest.java

```diff
@@ -522,4 +526,55 @@ void updateFailedWithWhiteSpace(String name) {
         assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST);
         assertThat(errorResponse.getMessage()).isEqualTo("회원명에 공백은 포함될 수 없습니다.");
     }
+
+    @DisplayName("문제집명이 포함된 문제집의 작성자들을 가져온다. - 성공")
+    @Test
+    void findAllUsersByWorkbookName() {
+        서로_다른_관리자의_여러개_문제집_생성_요청(ADMIN_WORKBOOK_REQUESTS_WITH_TAG, ADMINS);
+
+        // given
+        final String workbookName = "Java";
+        final HttpResponse response = request()
+                .get("/api/users?workbook=" + workbookName)
+                .build();
+
+        // when
+        final List<UserFilterResponse> userResponses = response.convertBodyToList(UserFilterResponse.class);
+
+        // then
+        assertThat(userResponses).hasSize(3);
+        assertThat(userResponses)
+                .extracting(UserFilterResponse::getUserName)
+                .containsExactlyInAnyOrder("user1", "user2", "user3");
```

Comment:
```suggestion
        assertThat(userResponses)
                .hasSize(3)
                .extracting(UserFilterResponse::getUserName)
                .containsExactlyInAnyOrder("user1", "user2", "user3");
```
이건 그냥 꿀팁 공유인데 hasSize는 이런식으로 메서드 체이닝을 할 수 있더라구요!

Reply:
이것도 소나린트가 훈수해주더라구요!! 근데 먼가 분리해주고 싶어서 분리했어여!!